### ISSUE TITLE
Delays in the approval process

### ISSUE THREAD

**Issue by federico-terzi** [federico-terzi/espanso-hub#64]
Hey guys,

I'm really sorry the review process is currently taking a lot right now, we are busy refactoring the internals of espanso for the new 2.0 release. 
There is a good chance some things will change for packages as well, so I'm leaving them on hold for now, so that we can "batch-process" them once the specs are ready.

Thank you for your understanding :)

**Comment by IdiosApps** [comment]
For outdated packages, Espanso notes: 
```
Index updated!
Warning: the requested package is hosted on an external repository:
...
espanso install misspell-en_US --external
```

**Would it be possible for unapproved packages to be installed via direct URL,** e.g. 
`espanso install https://github.com/IdiosApps/espanso-pinyin-tones --external` 

I tried this and got:
```
Allowing external repositories
Updating package index...
Index updated!
Package not found
```
So it appears that the `--external` flag is only for the use-case where an approved package has been updated.

This could be quite a useful feature - if there's no time to approve new packages, users can still install them. Even if the repo goes stale for years, a community can still build around it.

A compromise would be to whitelist only Github URLs, as they're open source. Users who enter the full link should be aware that it hasn't been approved by Espanso, and that they should ideally check the source code. Perhaps Espanso could ask for user confirmation as an extra protection.

**Comment by IdiosApps** [comment]
Ignore most of the above - you were obviously thinking ahead!

I was able to install my unapproved package with:
`espanso install -e espanso-pinyin-tones https://github.com/IdiosApps/espanso-pinyin-tones`

I don't remember seeing such a note in documentation however. 
I guess you prefer the espanso package system to be used, so that's fair!

**Comment by federico-terzi** [comment]
Hey guys!

First of all, sorry for the late update! The new v2 took a long time to complete, so I had to temporarily focus my efforts there :)
I've added most of the pending packages to the new [hub repository](https://github.com/espanso/hub), which will be the new standard location for "verified" packages.

A couple of notes:
* This new hub is only accessible through Espanso v2, so if you are using the legacy Espanso version (0.7.3), you 
won't be able to install these packages yet. 
After upgrading, you'll be able to install the packages with the usual `espanso install <package_name>`
* The hub at [hub.espanso.org](https://hub.espanso.org) is still showing the old repository with the legacy format, so this package is not visible yet.
We are working hard to create a new (and improved!) Hub. Once that will be ready, your package will automatically show there :)
* The v2 uses a slightly different package format, so I converted the packages before adding them to the hub.
The v2 package documentation is not yet there, but I'm working hard to get it up to speed soon :)

Cheers :)
